source/administration/custom-terms-of-service.rst

```diff
@@ -48,3 +48,11 @@ Why isn't this feature in Team Edition for GDPR compliance?
 Terms of service is presented to users on login and account creation, and available to users at all times in the link specified at **System Console > Customization > Legal and Support > Terms of Service link**.
 
 This feature is intended to meet compliance requirements for large Enterprise companies.
+
+Known Issues
```

Comment:
Propose we change this to an FAQ similar to https://docs.mattermost.com/administration/compliance-export.html#why-are-compliance-exports-beta

```
Why are custom terms of service beta?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

This feature is labelled as beta while we verify the quality of the feature in various scenarios. Known issues include:

- Login fails when custom terms of service is enabled and MFA is enforced.
- Terms of service re-appears on page refresh even if it was already accepted.
- Terms of service text disappears in the System Console on save until next refresh.
- Terms of service loads immediately following an update to terms, instead of after login or a page refresh.